src/components/career-form/career-form.tsx

```diff
@@ -112,102 +143,72 @@ export const CareerForm: React.FC<CareerFormProps> = (props) => {
-              name="first_name"
-              label="Vorname"
-            />
-          </GridItem>
+          <CareerTextFields register={register} errors={errors} />
 
-          {/*Last Name*/}
-          <GridItem xs={12} md={6}>
-            <InputField
-              required={true}
-              inputRef={register({ required: 'Dein Nachname fehlt' })}
-              error={errors.last_name}
-              name="last_name"
-              label="Nachname"
-            />
-          </GridItem>
-
-          {/*E-Mail*/}
-          <GridItem xs={12} md={6}>
-            <InputField
-              required={true}
-              inputRef={register({
-                required: 'Deine E-Mail fehlt',
-                pattern: {
-                  value: SIMPLE_EMAIL_PATTERN,
-                  message: `Irgendwas stimmt an dieser E-Mail nicht`,
-                },
-              })}
-              error={errors.email}
-              name="email"
-              label="E-Mail-Adresse"
-            />
-          </GridItem>
-
-          {/*Location*/}
-          <GridItem xs={12} md={6}>
-            <InputField
-              inputRef={register()}
-              error={errors.location}
-              name="location"
-              label="Aktueller Wohnort"
-            />
-          </GridItem>
-
-          {/*Available From*/}
-          <GridItem xs={12} md={6}>
-            <InputField
-              inputRef={register()}
-              error={errors.available_from}
-              name="available_from"
-              label="Verfügbar ab"
-            />
-          </GridItem>
-
-          {/*Salary Expectations*/}
-          <GridItem xs={12} md={6}>
-            <InputField
-              inputRef={register()}
-              error={errors.salary_expectations}
-              name="salary_expectations"
-              label="Gehaltsvorstellung"
-            />
-          </GridItem>
-
-          {/*Cover Letter*/}
+          {/*File-Upload*/}
           <GridItem>
-            <InputField
-              required={true}
-              inputRef={register({ required: 'Dein Anschreiben fehlt' })}
-              error={errors.message}
-              name="message"
-              label="Anschreiben"
-              type={'text-area'}
-            />
+            <FileInput
+              setValue={setValue}
+              clearErrors={clearErrors}
+              name="documents"
+              register={register}
+              selectedFiles={selectedFiles}
+              error={errors.documents}
+            >
+              <>
+                {(!selectedFiles || selectedFiles.length === 0) && <Upload />}
+                <div>
+                  Drop files to upload or <span>browse</span>
+                </div>
+              </>
+            </FileInput>
+            <FormError error={errors.documents} />
           </GridItem>
 
-          {/*CV File */}
+          {/*File-Review*/}
+          {selectedFiles &&
```

Comment:
For me file review + file input could be a single component "Document Upload" because I will never use the parts in isolation so I'm happy to encapsulate them and hide the logic from the form component (similar to the text fields component you created within this PR).